FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, from: response))
+
+			default:
```

Comment:
`result` has only two cases: `.success` and `.failure`. Using `default` here can be confusing whereas using `.failure` is more explicit and strict. 

If in future `Result` type is required to have tree cases or more (`success`, `failure`, `otherCase`) then having only `.success` and `.failure` cases in switch will cause compilation error and will force the developer to fix it. On the other hand having `.success` and `default` will run smoothly and potentially can lead to a bug or to a mishandled case.

Please replace `default` case with `.failure`

```suggestion
			.failure:
```